net-libs/libtelnet/libtelnet-0.23.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils autotools
+
+DESCRIPTION="Simple RFC-complient TELNET implementation as a C library"
+HOMEPAGE="https://github.com/seanmiddleditch/libtelnet"
+SRC_URI="https://github.com/seanmiddleditch/libtelnet/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	_elibtoolize
```

Comment:
This looks a bit odd. I reckon you can get away with simply `eautoreconf` here. We avoid calling these guys individually if we can help it.

Reply:
Yeah, I was unsure about this, `eautoreconf` works just fine :)

Reply:
Perfect. Is it actually needed in the first place at all btw?

Reply:
I think so, nothing is actually compiled if it's not called